source/process/working-at-mattermost.rst

```diff
@@ -0,0 +1,73 @@
+9. **Global perspective:** Develop outstanding software in a multi-lingual, multi-cultural community. 
+10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries. 
+
+----------------------------------------------------------
+Working Remotely 
+----------------------------------------------------------
+
+While we're headquartered in Palo Alto, California, like many open source companies, a large portion of Mattermost, Inc. works remotely from around the world. 
+
+This gives us tremendous advantages: 
+
+1. **Access to top talent.** We can hire from anywhere.
+2. **More comfort.** People have 100% control of their surroundings. 
+3. **No commute.** Removing a 30-60 minute commute saves 250-500 hours a year. 
+4. **Time zone coverage.** It's easier to support customers any time of day, without being up at odd hours.
+5. **Results-focused culture.** Output matters more than hours at a desk. 
+
+Running a distributed, high performance organization 
+
+1) Work from anywhere with good Internet
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Working from anywhere means more time with loved ones, no commute, and controlling everything about your workspace. 
+
+The one requirement is having a strong internet connection for occassional video or audio conferences. Ethernet connections are preferred over Wifi, since Wifi can have interference. 
+
+2) Be explicit 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you send a message, you take someone's full attention, so get across the relevant information as soon as you can.
+
+Don't say "Are you free to talk?", instead say "Catch up on ulimit issue Customer X is having?"
+
+Likewise, over communicate in writing when you can. The more people can read, the faster they can become effective. 
+
+3) Follow through 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When working on problems, make sure to be clear about next steps, and either follow through or create a ticket to track the work. 
+
+4) Hang out 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We have `warm-ups <https://docs.mattermost.com/process/meetings.html#warm-ups>`_ in some meetings to learn more about each other. It's part of a culture where we get to know each other. 
+
+Find time after meetings and video conferences to get to know your team mates. Ask them about their weekends and how their life is outside of work.
+
+When you meet in person at off-sites and events, build on those relationships.  
+
+5) Be the change you want to see
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When something's not clearly documented, take the intiative to figure out what's going on and update the documentation for the people who come after you. 
+
+"Learn, Master, Teach" is a model we use to pass on training after you've been trained to help grow everyone at the company. 
```

Comment:
Maybe add a few more details on Learn Master Teach or link to other documentation to explain what it is